10-angular/introduction_to_angular/readme.md

```diff
@@ -54,7 +55,6 @@ Take a look at [this quick overview of angular](http://www.tutorialspoint.com/an
 
 T&T - Some points to think about (5)
 - What are some features or characteristics of Angular?
```

Comment:
Suggest more questions to frame what they're looking for
- What are some of the advantages of Angular? Disadvantages?